src/vm/arm64/asmhelpers.S

```diff
@@ -1220,52 +1220,52 @@ NESTED_END StubDispatchFixupStub, _TEXT
 // ------------------------------------------------------------------
 // void* JIT_GetSharedNonGCStaticBase(SIZE_T moduleDomainID, DWORD dwClassDomainID)
 
-    LEAF_ENTRY JIT_GetSharedNonGCStaticBase_SingleAppDomain
+LEAF_ENTRY JIT_GetSharedNonGCStaticBase_SingleAppDomain, _TEXT
     // If class is not initialized, bail to C++ helper
     add x2, x0, #DomainLocalModule__m_pDataBlob
-    ldrb w2, [x2, w1]
```

Comment:
This is an invalid instruction form.  Given that both arguments are 64 bit, I assumed the following form was what was intended.